ext/tags.go

```diff
@@ -73,9 +73,12 @@ var (
 	// PeerHostname records the host name of the peer
 	PeerHostname = stringTagName("peer.hostname")
 
-	// PeerHostIPv4 records IP v4 host address of the peer
+	// PeerHostIPv4 records IP v4 host address of the peer as an uint32 value, keep this for backward compatibility
 	PeerHostIPv4 = uint32TagName("peer.ipv4")
 
+	// PeerHostIPv4String records IP v4 host address of the peer as a .-separated tuple. E.g., "127.0.0.1"
+	PeerHostIPv4String = stringTagName("peer.ipv4")
```

Comment:
don't know if it's worth it, but it occurred to me that instead of a separate const we could define a new dedicated type for ipv4 that would have both Set(uint32) and SetString(string) methods. This would keep it both backwards compatible and keep the single public constant name.